src/page/personal-information.page.ts

```diff
@@ -41,23 +44,35 @@ export class PersonalInformation {
       .first()
       .click();
   }
-
+  private async okAlert(): Promise<void> {
+    await browser.wait(ExpectedConditions.alertIsPresent(), 10000);
+    await browser.switchTo().alert().accept();
+  }
   public async fillForm(data: PersonalInfo): Promise<void> {
     await this.firstName.sendKeys(data.firstName);
     await this.lastName.sendKeys(data.lastName);
     await this.selectRadio(this.sex, data.sex);
     await this.selectRadio(this.experience, String(data.experience));
     await this.selectRadio(this.profession, data.profession[0]);
+    await this.profile.sendKeys(resolve(data.file));
     await this.selectRadio(this.tools, data.tools[0]);
     await this.continent.sendKeys(data.continent);
-    await browser.sleep(5000);
+  }
+
+  public async submit(): Promise<void> {
     await this.button.click();
+    await this.okAlert();
   }
 
   public async confirm(): Promise<string> {
-    await browser.wait(ExpectedConditions.alertIsPresent(), 10000);
-    const alert = await browser.switchTo().alert();
-    await alert.accept();
     return await this.title.getText();
   }
+
+  public async getFiles(): Promise<number> {
+    const numberOfFiles: number = await browser.executeScript(
+      ' return arguments[0].files.length;',
+      this.profile
+    );
+    return numberOfFiles;
```

Comment:
Is it possible to get the name of the files instead of the number, then in the assertion you verify that correct file was uploaded using the name instead of using the number of files